studio/components/interfaces/Database/Extensions/Extensions.tsx

```diff
@@ -49,13 +50,21 @@ const Extensions: FC<Props> = ({}) => {
             onChange={(e) => setFilterString(e.target.value)}
             icon={<IconSearch size="tiny" />}
           />
-          {!canUpdateExtensions && (
+          {!canUpdateExtensions ? (
             <div className="w-[500px]">
               <InformationBox
                 icon={<IconAlertCircle className="text-scale-1100" strokeWidth={2} />}
                 title="You need additional permissions to update database extensions"
               />
             </div>
+          ) : (
+            <Link passHref href="https://supabase.com/docs/guides/database/extensions">
```

Comment:
We should just use /docs/guides/database/extensions so it links to the correct env